lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskRuntime.cs

```diff
@@ -83,6 +85,20 @@ public TaskRuntime(IInjector taskInjector, string contextId, string taskId, Hear
                 // do not rethrow since user is not required to provide name client
             }
 
+            _driverConnectionMessageHandler = new Lazy<IDriverConnectionMessageHandler>(() =>
```

Comment:
The reason I am using `Lazy` is because this variable not necessarily going to be used. I'm not sure how costly it is to call `_injector.GetInstance()`, but I thought it would be nice if the instantiation could be avoided entirely if the user doesn't implement an `IDriverConnectionMessageHandler`.